benchmark_runner/common/oc/oc.py

```diff
@@ -433,6 +442,17 @@ def save_pod_log(self, pod_name: str, database: str = ''):
             self.run(f"{self.__cli} logs -n {environment_variables.environment_variables_dict['namespace']} {pod_name} > {output_filename} ")
         return output_filename
 
+    def describe_pod(self, pod_name: str, namespace: str = ''):
+        """
+        This method describe pod into log
+        :param pod_name: pod name with uuid
+        :param namespace: namespace
+        :return: output_filename
+        """
+        output_filename = os.path.join(self._run_artifacts, f'describe-{pod_name}')
+        self.run(f"{self.__cli} describe pod -n {namespace} {pod_name} > {output_filename} ")
```

Comment:
Quoting and trailing whitespace:

```
self.run(f"'{self.__cli}' describe pod -n '{namespace}' '{pod_name}' > '{output_filename}'")
```

Reply:
I have added it only to '{output_filename}', for 'oc' it will cause syntax's issue and also we pass this parameter.